sourcegraph/sourcegraph
# Add IDE CTA pings

**Important:** This PR builds upon #29966, merge that first, then rebase!

- Added a few lines to the site admin "pings" page, plus rationalized BExt ping description as well
- Added notes to pings.md, plus rationalized BExt ping description as well
- Added IDE CTA data to CTAUsage in types.go
- Added IDE CTA-related SQL to CTA query, plus rationalized existing query lines and renamed a few variables to accommodate for the larger amount of data
- Updated and ran the related test—it works as intended.

## Test plan

- Run `sg start`
- Check the latest ping on the site admin page to see if the new data is included there
